guidelines/terms/21/single-pointer.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn>single pointer</dfn></dt>
 <dd>
-   	
-   <p>pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures</p>
-  				
+   <p>an input – such as a mouse, stylus, or finger on a touch screen – that only targets a single point on the screen (as opposed to multi-pointer/multi-touch scenarios)</p>
```

Comment:
but this definition does not try, nor need to, define what a path-based gesture is or isn't. this is the definition purely of what a "single pointer" is. not what a gesture is. which was the whole problem initially where the definition completely messed up the subject of the definition and tried to define interactions

Reply:
the problem of "what is a path-based gesture" is one that needs to be disambiguated/explained/patched in 2.5.1 Pointer Gestures and (now that we have it) 2.5.7 Dragging Movements.

and to be clear, I absolutely agree that the whole "intermediate point" thing was an ugly, ugly fudge that was used to retrospectively make 2.5.1 apply to things that we only later realised 2.5.1 wasn't actually covering, even though we thought it did. it was crowbarred into the understanding for 2.5.1 because it was too late once it was realised to go back to the drawing board and change/expand the normative SC wording... and now that we *also* have 2.5.7 there's even more confusion (as potentially now certain types of sliders/interactions would fall under just 2.5.7, or both 2.5.1 and 2.5.7, and it often looks arbitrary which one's which)

Reply:
Yes, makes sense to separate the interaction. I think the definition is fine. Will try to tackle the interaction bit separately (even though I fear there is little appetite for changing path-based gesture definition now that we also have 2.5.7).

Reply:
>  I fear there is little appetite for changing the path-based gesture definition now that we also have 2.5.7

https://en.wikipedia.org/wiki/Appetite_for_Destruction

but seriously, i think the actual existence (now) of 2.5.7 fives us more leverage to perhaps remove the fudges that were made in 2.5.1